### ISSUE TITLE
NPM run start erasing edits and reverting to original file

### ISSUE THREAD

**Issue by scottpayne93122** [StartBootstrap/startbootstrap-resume#268]
Hello,

If I close terminal and run "NPM start" more than once my edits are not saved and everything reverts back to the original template. Can someone point me in the right direction as to why this is happening?

I'm mainly editing the .html file in the dist directory but everything is reverted each time I run "NPM start"

This hasn't happened to me on other projects before. I'm wondering how this is happening so I can avoid it in the future. 

Thank you!

**Comment by Ali-Shaikh** [comment]
@scottpayne93122 You should edit the startbootstrap-resume/src/pug/index.pug file and not the file in the dist directory. Becuase when you run 'npm start' it will build the project from src and replace everything that is in the dist folder.

I hope this helps.

**Comment by bckelley** [comment]
> You should edit the startbootstrap-resume/src/pug/index.pug file

I agree if you edit the pug file  and not the html file your settings are saved.

this issues should be closed
